Package.swift

```diff
@@ -324,7 +351,7 @@ let package = Package(
         .package(
             name: "AWSSwiftSDK",
             url: "https://github.com/awslabs/aws-sdk-swift",
-            from: "0.2.3"
+            .exact("0.2.3")
```

Comment:
Why are we changing it to `.exact()` here?
Making a suggestion to revert to `from`. If there's a good reason to stick with `.exact` feel free to ignore the suggested change.
```suggestion
            from: "0.2.3"
```

Reply:
I changed it to `.exact` as it was pulling`0.2.4` which removed changes in configuration code from `async`. Is that the case going forward with `aws-sdk-swift`?

Reply:
In that case, we should go back to `from`. The changes in 0.2.4 (sync init overloads) are needed by other plugins.